jazelly/FinetuneLLMs
# Dockerize trainer

### Functional test steps

1. Run `docker compose build; docker compose up -d` and it should take about 6 minutes to build
2. Run `docker compose exec trainer bash`
3. Then run `python manage.py help`. It should show you available commands

Fixes: https://github.com/jazelly/FinetuneLLMs/issues/11
